doc/thing/thing-requirements.rst

```diff
@@ -0,0 +1,147 @@
+Requirements
+============
+
+Linux based OS
+--------------
+
+The actual version of KNoT Zephyr SDK is only available for Linux based systems.
+You may consider using a Virtual Machine running a Linux distribution on other systems.
```

Comment:
Please, also suggest the usage of Docker for it. We should use Docker prior than use VMs

Reply:
Done